src/user/device/list.js

```diff
@@ -14,5 +14,5 @@ const { DEFAULT_REQ_OPTS } = require('../../utils/defaultReqOpts')
  */
 
 module.exports = serializeAxiosResponse(
-  ({ userId }) => API.get('user', `${userId}/devices`, DEFAULT_REQ_OPTS),
+  ({ userId, queryStringParameters }) => API.get('user', `${userId}/devices`, { queryStringParameters, ...DEFAULT_REQ_OPTS }),
```

Comment:
if we go by convention this should be (userId, { queryStringParameters}) but that would make it a breaking change so I'm keeping it as is